### DISCUSSION TITLE
如何在父节点下找到所有的子节点？

### DISCUSSION THREAD

**Discussion by xdJumpING** [oflg/TiddlyMemo#2] [Q&A]
Tiddlywiki里似乎没有文件夹的概念。
在某个节点下可新建子节点，子节点里可找到父节点的标签链接，但父节点里看不到子节点。请问有什么办法可以看到吗？

**Comment by oflg** [comment]
在笔记里写上

`<$list filter="[tag[父节点标题]]"/>`

参照：https://tiddlywiki5.vercel.app//#ListWidget
